packages/js/src/dashboard/widgets/site-kit-setup-widget.js

```diff
@@ -134,43 +132,44 @@ export const SiteKitSetupWidget = ( { dataProvider, remoteDataProvider } ) => {
 				</DropdownMenu.ButtonItem>
 			</DropdownMenu.List>
 		</DropdownMenu>
-		<div className="yst-flex yst-justify-center yst-mb-6 yst-mt-4"><YoastConnectSiteKit width="252" height="60" /></div>
+		<div className="yst-flex yst-justify-center yst-mb-6 yst-mt-4">
+			{ isSiteKitConnectionCompleted  ? <YoastConnectSiteKitSuccess width="252" height="60" />
+				: <YoastConnectSiteKit width="252" height="60" /> }
+		</div>
 		<Stepper steps={ steps } currentStep={ currentStep }>
 			{ steps.map( ( label, index ) => (
 				<Stepper.Step
 					key={ label }
 					isActive={ currentStep === index }
-					isComplete={ dataProvider.getStepsStatuses()[ index ] }
+					isComplete={ currentStep > index || isSiteKitConnectionCompleted }
 				>
 					{ label }
 				</Stepper.Step>
 			) ) }
 		</Stepper>
 		<hr className="yst-bg-slate-200 yst-my-6" />
-		<Title size="2">{ __( "Expand your dashboard with insights from Google!", "wordpress-seo" ) }</Title>
+		<Title size="2">{ isSiteKitConnectionCompleted ? __( "You’ve successfully connected your site with Site Kit by Google!", "wordpress-seo" ) : __( "Expand your dashboard with insights from Google!", "wordpress-seo" ) }</Title>
```

Comment:
This line is mighty long. [We have an linting rule exception for string and template literals](https://github.com/Yoast/wordpress-seo/blob/6b38fd046130b74720ada4c0502141d4829c974f/packages/js/eslint.config.mjs#L69). But that is a bit too friendly towards lines that include those, but can still be easily split.

```suggestion
		<Title size="2">
			{ isSiteKitConnectionCompleted
				? __( "You’ve successfully connected your site with Site Kit by Google!", "wordpress-seo" )
				: __( "Expand your dashboard with insights from Google!", "wordpress-seo" )
			}
		</Title>
```